### ISSUE TITLE
Tests currently failing for g++ with c++11

### ISSUE THREAD

**Issue by dbuse** [sommer/veins#100]
Since ref aa75b3 the tests no longer compile.
See https://github.com/sommer/veins/commit/aa75b3250b69ba1d0df9f7e40cf73c8bccd5db3b#r31379876 for details.

**Comment by sommer** [comment]
Unable to reproduce, but might be fixed in 9385cc6b. Could you please check?

**Comment by dbuse** [comment]
That worked for one section, but the tests are still failing at a similar place:
```
SignalRepresentation.cc: In function 'void ____C_A_T_C_H____T_E_S_T____177()':
SignalRepresentation.cc:1380:9: error: converting to 'std::tuple<double, double, double>' from initializer list would use explicit constructor 'constexpr std::tuple< <template-parameter-1-1> >::tuple(_UElements&& ...) [with _UElements = {int, double, float}; <template-parameter-2-2> = void; _Elements = {double, double, double}]'
         };
         ^
```

To reproduce, compile both veins and veins_catch with g++, e.g., under Ubuntu 16.04

**Comment by sommer** [comment]
Successfully reproduced with g++ 5.4

**Comment by sommer** [comment]
Addressed in 23ef58aa
